Client/Frontend/Settings/AppSettingsOptions.swift

```diff
@@ -407,6 +407,24 @@ class ExportBrowserDataSetting: HiddenSetting {
     }
 }
 
+class EnableActivtyStreamSetting: HiddenSetting {
```

Comment:
Suggest wrapping this up into a `FeatureSwitchSetting`, extending `BoolSetting` and have `EnableActivtyStreamSetting` derive from that.

I expect you'll need to refactor the line: 

```swift
control.isOn = prefs.boolForKey(prefKey) ?? defaultValue
```

and 

```swift
    @objc func switchValueChanged(_ control: UISwitch) {
        prefs.setBool(control.isOn, forKey: prefKey)
        settingDidChange?(control.isOn)
    }
```